examples/book-checkout.yaml

```diff
@@ -0,0 +1,57 @@
+
+'@context':
+  - https://git.io/vf-examples-jsonld-context
+  - library: https://library.example/
+    bob: https://bob.example/
+
+
+'@id': rgh:valueflows/valueflows/master/examples/book-checkout.yaml
+'@graph':
+
+  # resource before
+  - '@id': library:21f361a6-2375-46bb-b192-c21b5ba833bf
+    '@type': EconomicResource
+    skos:note: Valueflows cookbook
+    currentQuantity:
+      qudt:unit: unit:Number
+      qudt:numericValue: 1
+
+  - '@id': library:f47064eb-7120-4b31-b882-770165901fe6
+    '@type': Agreement
+    skos:note: renting Valueflows cookbook to Bob
```

Comment:
This is interesting, missed the Agreement earlier.  Curious how would you combine that with the other side of the transfer (the increment on bob's side when he checks out the book), which would have a Process?

Reply:
I don't know if we need to assume any 'other side'. Bob can just have that commitment but don't keep an inventory. Also if inclusion in inventory implies ownership, Bob might just increment the same resource `library:21f361a6-2375-46bb-b192-c21b5ba833bf` at his place. I wouldn't see it as some 'other side' of the transfer of *use* rights to the book, R&R already happened on check-out from library, recording now location of the same book on his bookshelf doesn't have anything to do with transfer any more. He could as well just keep it in his backpack and return couple of days later, fulfilling the commitments and transferring the *use* rights back to library but never checking it in and out again from anywhere else.

Reply:
No, we don't need to assume it of course.  But we also need to know how to handle it if it is needed, so I'd like to "assume" it for purposes of at least one example.  And in particular, the "mostly duplication" of the give/receive or decrement/increment in a process that is basically the same event content with a different agent's resource, I think we need to experience first hand to see if it is acceptable.